plugin/src/main/java/dev/regadas/trino/pubsub/listener/PubSubEventListenerConfig.java

```diff
@@ -2,12 +2,14 @@
 
 import static java.util.Objects.requireNonNull;
 
+import com.google.api.gax.batching.BatchingSettings;
 import com.google.auto.value.AutoBuilder;
 import com.google.pubsub.v1.TopicName;
 import dev.regadas.trino.pubsub.listener.Encoder.Encoding;
 import java.util.Map;
 import java.util.Optional;
 import javax.annotation.Nullable;
+import org.threeten.bp.Duration;
```

Comment:
We are using [java 17](https://github.com/regadas/trino-pubsub-event-listener/blob/main/.github/workflows/build.yml#L13), why not using `java.time`'s Duration?

Reply:
So that is only required because `com.google.api.gax.batching.BatchingSettings` uses that `Duration` type unfortunately.